Source/LinqToDB/Mapping/Nullability.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Reflection;
+
+namespace LinqToDB.Mapping
+{
+	static class Nullability
+	{
+		private static Lazy<bool> _isSupported = new(() => 
+		{
+#if NET461_OR_GREATER || NETSTANDARD2_0_OR_GREATER || NETCOREAPP2_0_OR_GREATER || NET5_0_OR_GREATER
```

Comment:
I believe .NET 5 targets include NETCOREAPP2_0_OR_GREATER.

Ref: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives

```suggestion
#if NET461_OR_GREATER || NETSTANDARD2_0_OR_GREATER || NETCOREAPP2_0_OR_GREATER
```

Reply:
Yeah, I believe that might actually be the case.
If you're 100% sure you can remove it, otherwise it doesn't hurt anyway.